### PR TITLE
RIA-2951 gradle version bumped to v6.5

### PR DESCRIPTION
**Pull request by tarun-palisetty**
### JIRA link (if applicable) ###
[RIA-2951](https://tools.hmcts.net/jira/browse/RIA-2951)

### Change description ###
Gradle version bumped to v6.5

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
```
Comment (raghera) [inline gradle/wrapper/gradle-wrapper.properties:5]: Think you've added Windows Line Feed chars at the end of the file.  Can you remove and stick to `LF` only please?

File: gradlew [hunk 2]
```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
Comment (raghera) [inline gradlew:31]: Any reason why you had to add these?  May cause an issue for some if not enough on their OS.

File: src/integrationTest/java/uk/gov/hmcts/reform/iacaseapi/component/testutils/fixtures/PreSubmitCallbackResponseForTest.java [hunk 3]
```diff
@@ -3,10 +3,12 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.Set;
+import lombok.Getter;
```
Comment (raghera) [inline src/integrationTest/java/uk/gov/hmcts/reform/iacaseapi/component/testutils/fixtures/PreSubmitCallbackResponseForTest.java:6]: Are these tests changed for the Gradle upgrade?  They seem unrelated.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (tarun-palisetty) [reply gradle/wrapper/gradle-wrapper.properties]: Now the CRLF character has been removed
